backend/capellacollab/projects/permissions/crud.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import sqlalchemy as sa
+from sqlalchemy import orm
+
+from capellacollab.projects import models as projects_models
+from capellacollab.users.tokens import models as tokens_models
+
+from . import models
+
+
+def create_project_token(
+    db: orm.Session,
+    project: projects_models.DatabaseProject,
+    token: tokens_models.DatabaseUserToken,
+    scope: models.ProjectUserScopes,
+) -> models.DatabaseProjectPATAssociation:
+    model = models.DatabaseProjectPATAssociation(
+        token=token, project=project, scope=scope
+    )
+    db.add(model)
+    db.commit()
+    return model
```

Comment:
The name of this is a bit misleading since it's a project token association and not a project token